src/SprykerEco/Zed/Econda/Dependency/Plugin/EcondaPluginInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Dependency\Plugin;
+
+use Generated\Shared\Transfer\BatchResultTransfer;
+use Generated\Shared\Transfer\LocaleTransfer;
+use SprykerEco\Zed\Econda\Business\Exporter\Writer\File\FileWriterInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+interface EcondaPluginInterface
```

Comment:
This name is too generic. Please be more specific.

Reply:
see suggestion in : https://github.com/spryker-eco/econda/pull/1/files#diff-6370d6728b905d928d1252bab2a6da50R80

Reply:
done